mlflow/tracking/client.py

```diff
@@ -493,17 +493,22 @@ def update_model_version(self, name, version, description=None):
                                                                 description=description)
 
     @experimental
-    def transition_model_version_stage(self, name, version, stage):
+    def transition_model_version_stage(self, name, version, stage, archive_existing_versions=False):
         """
         Update model version stage.
 
         :param name: Registered model name.
         :param version: Registered model version.
         :param stage: New desired stage for this model version.
+        :param archive_existing_versions: If this flag is set to ``True``, all existing model
+            versions in the stage will be automically moved to the "archived" stage. Only valid
+            when ``stage`` is ``"staging"`` or ``"production"`` otherwise an error will be raised.
```

Comment:
@andychow-db cc @sueann 
Fixed the description for `archive_existing_versions`. After making sure this is fine, I'll apply the change to other `transition_model_version_stage`s.